low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -54,7 +55,7 @@ def usage(prog: str):
             [ post_install --config [<global_config_url>] ]
             [ init --config [<global_config_url>] ]
             [ config --config [<global_config_url>] ]
-            [ test [sanity|alerts] ]
+            [ test [sanity|alerts|self_primary|self_secondary] --config [<global_config_url>] ]
```

Comment:
[ test --config [<global_config_url>] ] --plan [sanity|alerts|self_primary|self_secondary]

if --plan not given, Internally code should default to 'self_primary'